code/modules/mob/living/carbon/alien/humanoid/humanoid.dm

```diff
@@ -9,7 +9,6 @@
 	var/next_attack = 0
 	var/pounce_cooldown = 0
 	var/pounce_cooldown_time = 30
-	update_icon = 1

```

Comment:
i don't even know why this was here, aliens are hardcoded to call update_icons() on Life()